### PR TITLE
Add command line to install packages on Ubuntu 16.04

### PR DESCRIPTION
**Pull request by AxelNennker**
The names of the packages given in this file do not work out of the box.
I suggest to add the command line that installs the packages.

I also included the autoconf automake libtool packages to have it all in one go.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ libgcrypt-devel libselinux-devel libxml2-devel libxslt-devel make openldap-devel
 pcre-devel perl-XML-Parser perl-XML-XPath perl-devel python-devel rpm-devel swig \
 bzip2-devel
 ```
+On Ubuntu 16.04 the command to install these package is
+```code
+sudo apt-get install -y autoconf automake libtool make libdbus-1-dev libdbus-glib-1-dev libcurl4-openssl-dev libgcrypt20-dev libselinux1-dev libxslt1-dev libgconf2-dev libacl1-dev libblkid-dev libcap-dev libxml2-dev libldap2-dev libpcre3-dev python-dev swig libxml-parser-perl libxml-xpath-perl libperl5.22 python-dev libbz2-dev librpm-dev swig
+
```
Comment (matejak) [inline README.md]: This blank line in the code block below `apt-get install` seems distracting to me, does it have any purpose?

**Comment by matejak** [comment]
Thank you for your contribution, it is useful to have Ubuntu instructions up-to-date. Just for the reference, the autoconf/automake packages are required only if one is building from the repository (but I think that it is good to have them in the command).

File: README.md (hunk 1 above)
Comment (AxelNennker) [reply README.md]: Hm, not really. I thought there was a ``` to end the code block but the preview looked OK

**Comment by mpreisler** [comment]
@openscap-ci test this please

**Comment by jan-cerny** [comment]
Great! Thank you very much. I'm sorry for the huge delay.

**Pull request merged**